metecho/api/jobs.py

```diff
@@ -102,6 +102,9 @@ def epic_create_branch(
 def _create_branches_on_github(
     *, user, repo_id, epic, task=None, task_sha=None, originating_user_id=None
 ):
+    """
+    TODO: Rethink this function to accept projects as well, not only epics
+    """
```

Comment:
My hunch is that when a Task is created without an Epic we can just go to create a Task branch right away and skip Epic branch creation entirely since the "base" branch is the default project branch. For this function it means we can make `epic` nullable. What do you think @jgerigmeyer?

Reply:
@jerivas Yes, I think that's right. The project has a branch prefix (see the logic in `jobs.epic_create_branch`). When a task is created directly on a project, it should get a branch name the same way we're otherwise creating an epic branch: `f"{prefix}{slugify(task.name)}"`.

Reply:
Fixed in 2c22300e8a7d08378b20de83baf4a74ab7cb2a53